README.md

```diff
@@ -156,3 +157,14 @@ See the documentation folder
 The diagrams are saved as .jpg files, named per level of The C4 Model - as per https://c4model.com/ 
 
 The source code for these is in same named .drawio files and can be edited using https://app.diagrams.net/
+
+# Module 7 - Continuous Integration
+
+## To run the tests from docker
+```
+docker run --env-file ./.env.test todo-app:test
+```
+## New github workflow file
```

Comment:
While it's helpful to point this out, someone who understands what this means will probably know where to look, and someone who doesn't won't have much to work on here! Adding a quick sentence to explain that this dictates the pipelines that run on GitHub to test every branch might be quite worthwhile here